lib/views.rb

```diff
@@ -26,4 +28,59 @@ def menu
 		puts '                        C.Scores                      '
 		puts '__________________________________________________________'
         print "\n"
-end
\ No newline at end of file
+end
+def print_winner(game,dim,turn)
+	game.board.print_board(dim)
+    puts "Player#{turn} you won !"
+    game.score[turn.to_i  - 1] += 1
+end	
+def new_game(game) 
+  while true do
+  	dim = game.size_board
+    game.board.print_board(dim)
+    t = game.turn == "x" ? "1" : "2"   
+    puts "Player#{t} choose a number in 0 - #{dim*dim -1}  or Q to quit!"   
+    cell = gets.chomp
+    if cell.upcase == "Q"
+       puts "Quit"
+       break
+    elsif game.board.change_mat(cell.to_i,game.turn)
+        if game.board.check_winner(game.turn)
+          print_winner(game,dim,t)
+          break
+        elsif game.board.draw? 
+          game.board.print_board(dim)	
+          puts "its a draw"
+          game.score[2] += 1
+          break
+        end 
+        game.turn = game.switch_turn(game.turn)      
+    end     
+  end   
+end  
+#===================================================================================
```

Comment:
You can store all the next lines into a method. Name it `main`,  `start` of something similar